README.md

```diff
@@ -63,6 +63,6 @@ source before proceeding with the install.
 [slack]: http://slack.opengee.org
 [slack-img]: https://img.shields.io/badge/chat-on%20slack-ff69b4.svg
 [travis]: https://travis-ci.org/google/earthenterprise/builds
-[travis-img]: https://travis-ci.org/google/earthenterprise.svg?branch=master
+[travis-img]: https://travis-ci.org/google/earthenterprise.svg?branch=release_5.3.3
```

Comment:
Do we normally change the travis build like this? I thought we typically left it at master.

Reply:
I don't think we did it before, but it think this is a good practice to point to the correct travis-ci build... as long as we update it back again to master when merging to master.

Reply:
The release process doc says to change it in the release branch...